src/TConsole.cpp

```diff
@@ -2949,3 +2950,41 @@ void TConsole::setProfileName(const QString& newName)
         pC->setProfileName(newName);
     }
 }
+
+
+void TConsole::dragEnterEvent(QDragEnterEvent* e)
+{
+    if (e->mimeData()->hasUrls()) {
+        e->acceptProposedAction();
+    }
+}
+
+//https://amin-ahmadi.com/2016/01/04/qt-drag-drop-files-images/
+void TConsole::dropEvent(QDropEvent* e)
+{
+    QStringList accepted_types;
```

Comment:
For the drag and drop event, we probably shouldn't filter by images only - in case someone wants to make a script to read a text file from a drag and drop, for example?